src/clients/HubPoolClient.ts

```diff
@@ -537,8 +537,8 @@ export class HubPoolClient {
     });
     const timerStart = Date.now();
     const [currentTime, pendingRootBundleProposal, ...events] = await Promise.all([
-      this.hubPool.getCurrentTime(),
-      this.hubPool.rootBundleProposal(),
+      this.hubPool.getCurrentTime({ blockTag: searchConfig.toBlock }),
+      this.hubPool.rootBundleProposal({ blockTag: searchConfig.toBlock }),
```

Comment:
Should we add this blockTag to line 620 also in the `pooledTokens` call?

Reply:
Implemented here: https://github.com/across-protocol/sdk-v2/pull/228/commits/9136c7f1745e4d3b4b8a35cec075a0a1988ee980